src/controllers/ProfileController.js

```diff
@@ -26,15 +27,17 @@ export default class ProfileController {
     let person = await Person.findOne({ user: currentUser._id });
 
     if (!person) {
-      person =  Person.create({ ...body,  user: currentUser._id });
+      person =  await Person.create({ ...body,  user: currentUser._id });
```

Comment:
Yoo @nkpremices, Just a curious question does our Framework has a method similar to `find_or_create` in Django?